### PR TITLE
Fix problem with in-tree extension load

### LINKED ISSUE #3266: Lens5 - no more logs and console buttons when a pod is selected?
**Describe the bug**
image

**To Reproduce**
Click on a POD ... I used to have a console and logs to inspect a pod, no more since Lens 5.

**Environment (please complete the following information):**
- Lens Version: 5
- OS: windows
- Installation method (e.g. snap or AppImage in Linux): Windows setup

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #3266

Reverts changes in #3261 and includes a fix from #3256

### CODE REVIEW

**Comment by jakolehm** [comment]
Tested & verified by doing a local build and `.dmg` install:
image

**Review by msa0311 — APPROVED** [review]

**Comment by jim-docker** [comment]
Did you test on Windows or Ubuntu? #3266 says mac was OK, problem happens on Windows and Ubuntu.

**Comment by jakolehm** [comment]
> Did you test on Windows or Ubuntu? #3266 says mac was OK, problem happens on Windows and Ubuntu.

MacOS has the same problem. Did not test on Windows/Ubuntu because this shouldn't be OS specific issue (it's a version mismatch between Lens and extension).

File: src/extensions/extension-discovery.ts [hunk 1]
```diff
@@ -365,7 +365,13 @@ export class ExtensionDiscovery extends Singleton {
       let isCompatible = isBundled;
 
       if (manifest.engines?.lens) {
```
Comment (jim-docker) [inline src/extensions/extension-discovery.ts:367]: Is this code actually relevant to the bundled extensions? If so should we skip this if the extension is bundled?
```
if (!isBundled && manifest.engines?.lens) {
```

Comment (jakolehm) [reply src/extensions/extension-discovery.ts:367]: You are right, we could skip the check for bundled extensions.

**Pull request merged**